scripts/delayed.js

```diff
@@ -46,7 +46,8 @@ const pardotForm = document.querySelector('.pardot-form.block');
 if (campaignMeta && businessUnitMeta && pardotForm) {
   const campaignId = campaignMeta.getAttribute('content');
   const businessUnitId = businessUnitMeta.getAttribute('content');
-  const pardotUrl = pardotUrlMeta && pardotUrlMeta.getAttribute('content');
```

Comment:
IMO, this doesn't have to be an editable metadata element. 'contains' fails if no metadata is specified as well.
Can you take the hostname from Pardot Form block configured submit action?